pkg/controller/metrics/metrics.go

```diff
@@ -105,6 +106,31 @@ var (
 			Buckets: []float64{60, 90, 180, 300, 600, 1200},
 		},
 		[]string{"cluster_version", "platform", "cluster_pool_namespace", "cluster_pool_name"})
+	// MetricStoppingClusters is a prometheus metric that tracks the transition time for currently stopping clusters
+	MetricStoppingClusters = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "hive_cluster_deployments_stopping",
+			Help:    "Distribution of the length of transition time for clusters currently stopping",
+			Buckets: []float64{30, 60, 180, 300, 600},
+		},
+		[]string{"cluster_deployment", "platform", "cluster_version", "cluster_pool_namespace"})
```

Comment:
We need the CD namespace as well as its name

And we need the ClusterPool name as well as its namespace.

(I could get behind making both of these a `*_namespacedname` in theory, though that doesn't match what we're doing for other metrics.)

(Same below)